src/utils/constraints-utils.ts

```diff
@@ -0,0 +1,93 @@
+import * as snarkjs from "snarkjs";
+// @ts-expect-error: No type definitions available for the "r1csfile" package
+import * as r1csfile from "r1csfile";
+// @ts-expect-error: No type definitions available for the "@iden3/binfileutils" package
+import * as binfileutils from "@iden3/binfileutils";
```

Comment:
Do we have this package in the deps?

Reply:
no, but they are inside snarkjs dependencies

Reply:
By why do we need to rely on some transient dependencies?